### ISSUE TITLE
Convert to markdown

### ISSUE THREAD

**Issue by aaren** [aaren/notedown#5]
It might be useful to be able to convert a notebook back into markdown.

We can already do this using nbconvert:

```
ipython nbconvert notebook.ipynb --to markdown
```

There are two issues with this:
1. code cells are indented, no option for fence
2. outputs are converted to figures

Solution:

Provide a simple markdown template with notedown that allows
1. switching to fenced code blocks
2. stripping outputs

We then have a possible workflow of

```
edit markdown... 
convert to notebook... 
play with / edit notebook... 
convert to markdown... 
edit markdown
```

Basically, would let us roundtrip through the notebook format and preserve edits made to the text / code of notebooks.

Simple test:

```
notedown input.md > input.ipynb
notedown input.ipynb > output.md
diff input.md output.md
```

test strip outputs:

```
notedown input.md > input.ipynb
runipy input.ipynb output.ipynb
notedown output.ipynb > output.md
diff input.md output.md
```
